configuration/developing-community-health-applications.md

```diff
@@ -520,125 +529,122 @@ To build your targets into your app, you must compile them into app-settings, th
     medic-conf --local compile-app-settings backup-app-settings upload-app-settings 
 
 
-## Contacts
-In the `app_settings.contact_summary` you can write a script to output fields for the contact info pane and help decide which reports should be able to be filed against a contact. The script is evaluated as JavaScript so all the standard language features are available. To make this easier to write and maintain, a declarative format is also available. In the `contact_summary.templated.js` file you specify three variables: `context`, `fields`, and `cards`.
+## Contact Summaries
+In the `contact-summary.templated.js` file, write a script to control how a contact's profile page will appear by defining three outputs: `context`, `fields`, and `cards`.
```

Comment:
We should add an intro about Contact Pages, like we do for Tasks and Targets. Also, I think we should shift away from "write a script" since we are describing the declarative format in the sections below, even if we don't say it directly.

```suggestion
Health workers can easily access and manage the families and people they serve as profile pages in the People tab. These contact profiles display basic information about the contact along with their history and upcoming tasks. You can control how a contact's profile page will appear and behave by defining these three outputs in the `contact-summary.templated.js` file: `context`, `fields`, and `cards`.
```